module_4/framework/.phpcs.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
@Hlopec Only one linter is used. PHP Code Sniffer probably was a good tool. Sometimes, it is used together with PHP-CS-Fixer, but I suggest using the second one only.